sky/sdk/lib/widgets/variable_height_scrollable.dart

```diff
@@ -14,11 +12,16 @@ class VariableHeightScrollable extends Scrollable {
   VariableHeightScrollable({
     String key,
     this.builder,
-    this.token
-  }) : super(key: key);
+    this.token,
+    this.layoutState
+  }) : super(key: key) {
+    assert(layoutState != null);
```

Comment:
Good point again, I'd forgotten that I could just initialize layoutState in initState().